xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecTX3G.cpp

```diff
@@ -74,9 +75,9 @@ typedef struct
 CDVDOverlayCodecTX3G::CDVDOverlayCodecTX3G() : CDVDOverlayCodec("TX3G Subtitle Decoder")
 {
   m_pOverlay = nullptr;
-  m_textColor =
-      KODI::SUBTITLES::colors[CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(
-          CSettings::SETTING_SUBTITLES_COLOR)];
+  m_textColor = UTILS::COLORUTILS::ConvertHexToColor(
```

Comment:
I have kept the name of the previous class name because I thought there was a reason why this distinction was made,
but i think there is no problem use Color namespace